docs/src/manuals/inference/nonconjugate.md

```diff
@@ -0,0 +1,135 @@
+
+## Addressing Non-conjugacy with ExponentialFamilyProjection
+
+To overcome this limitation, `RxInfer` integrates with the `ExponentialFamilyProjection` package. This package re-projects non-conjugate relationships back into a member of the exponential family at the cost of some accuracy.
+
+!!! note
+    `RxInfer` supports non-conjugate inference for completeness, but be aware that inference execution times may increase significantly. This is because non-conjugate models require more complex computations, often involving sampling-based approximations.
+
+### Specifying Constraints
+
+The projection constraint must be specified using the `@constraints` macro. For example:
+
+```@example non-conjugacy-prior-likelihood
+using ExponentialFamilyProjection
+
+@constraints function non_conjugate_model_constraints()
+  # project variational posterior over `m` to `Beta`
+  q(m) :: ProjectedTo(Beta)
+  # project variational posterior over `p` to `Beta`
+  q(p) :: ProjectedTo(Beta)
+  # `m` and `p` are jointly independent
+  q(m, p) = q(m)q(p)
+end
+```
+
+These constraints specify that the posterior distribution for the hidden variable `m` must be re-projected to a `Beta` distribution to cover the region from `0` to `1`. The same applies to the variable `p`.  
+
+!!! note 
+    Note that the distribution specified in the `@constraints` does not need to match the distribution specified as a prior. For example, we could use a `Gamma` distribution as a prior and a `Beta` distribution as a posterior. The only requirement is that the support of the posterior distribution must be the same as or smaller than that of the prior.
+
+We also assume that `m` and `p` are jointly independent with the `q(m, p) = q(m)q(p)` specification.
+Dropping the assumption of joint independence would require initializing messages for `m` and `p` without guarantees of convergence.
+Read more about factorization constraints in the [Constraints Specification](@ref user-guide-constraints-specification) guide.
+
+The `ProjectedTo` structure is defined in the `ExponentialFamilyProjection` package. To fully explore its capabilities and hyper-parameters, we invite you to read the detailed [documentation](https://github.com/ReactiveBayes/ExponentialFamilyProjection.jl).
+
+### Initialization
+
+We also need to initialize the inference procedure due to the factorization constraints. Read more about initialization in the [corresponding section](@ref initialization).
+
+```@example non-conjugacy-prior-likelihood
+initialization = @initialization begin 
+  q(m) = Beta(1, 1)
+  q(p) = Beta(1, 1)
+end
+```
+
+### Running the Inference
+
+With everything set up, we can run the [inference](@ref user-guide-inference-execution) procedure:
+
+```@example non-conjugacy-prior-likelihood
+result = infer(
+  model = non_conjugate_model(),
+  data  = (y = data,),
+  constraints = non_conjugate_model_constraints(),
+  initialization = initialization,
+  returnvars = KeepLast(),
+  iterations = 5,
```

Comment:
Consider adding a brief explanation or visualization of how the inference results change with different numbers of iterations. This could help users understand the trade-off between computational time and inference accuracy. You could do this by running the inference multiple times with different iteration counts and plotting the results side by side.